Makefile

```diff
@@ -8,6 +8,7 @@ GOBUILD=$(GOCMD) build
 GOCLEAN=$(GOCMD) clean
 GOTEST=$(GOCMD) test
 BINARY_LOCATION=./haulage
+CONF_LOCATION=./config.yml
```

Comment:
I'm actually hesitant to do this, since the example config uses an interface that's specific to my local machine. I'm okay having it use the example config by default instead of erroring out, but I do think it should be clear the config is general. I think the best thing would be to have one of those post-install interactive terminal scripts that asks what interface you want to use and generates the correct config, but that's a long way in the future : )

Reply:
Hmm, thinking about the deb use case, I take back what I said above and I think it makes more sense to just call the file config.yml, but make it clear in a comment in the file that this is a first example.